components/NowPlayingList.js

```diff
@@ -25,7 +25,7 @@ export default function NowPlayingList() {
         <StyledUl>
           {nowPlayingMovies.map((movie) => (
             <StyledList key={movie.id}>
```

Comment:
This component looks like a SyledListItem, not the list itself. You should rename it to fit the intended usage.

Reply:
At the same time, you display only one ard in this item. Maybe you can add the styling to the Card component itself and remove the StyledListItem entirely?